boilerplate/openshift/golang-osd-operator/csv-generate/catalog-build.sh

```diff
@@ -2,16 +2,20 @@
 
 source `dirname $0`/common.sh
 
-usage() { echo "Usage: $0 -o operator_name -c saas-repository-channel" 1>&2; exit 1; }
+usage() { echo "Usage: $0 -o operator-name -c saas-repository-channel -r registry-image" 1>&2; exit 1; }
```

Comment:
`registry-image` for local testing? I thought about adding the same thing.

Reply:
Yes, this allowed me to remove hardcoding that uri so I could set the environment variable from my shell.

Reply:
I've put together a full guide for local testing [here](https://github.com/openshift/boilerplate/blob/eb067b23871d77cefb8eca1f7ea2c0f51cd1d719/boilerplate/openshift/golang-osd-operator/app-sre.md). Enjoy.